Cargo.toml

```
@@ -22,14 +22,13 @@ alloc = []
 arb = ["dep:quickcheck", "dep:rand", "dep:arbitrary"]
 scale-codec = ["dep:parity-scale-codec"]
 serde-codec = ["serde"] # Deprecated, don't use.
-serde = ["dep:serde", "dep:serde-big-array"]
+serde = ["dep:serde"]
 
 [dependencies]
 parity-scale-codec = { version = "3.0.0", default-features = false, features = ["derive"], optional = true }
 quickcheck = { version = "1.0.3", optional = true }
 rand = { version = "0.8.5", optional = true, features = ["small_rng"] }
 serde = { version = "1.0.116", optional = true, default-features = false, features = ["derive"] }
```

```suggestion
serde = { version = "1.0.116", optional = true, default-features = false }
```

I think we can drop that feature now!

---

Good catch!